### PR TITLE
[REF] doc: howtos - Remove <data> tag from xml files.

### PR DESCRIPTION
**Pull request by moylop260**
Based on comment from commit message:
 - https://github.com/odoo/odoo/commit/7c0e598c97cc48d8e55688c99db6d11ee23d9e46

### CODE REVIEW

**Comment by moylop260** [comment]
@mart-e What do you think?

File: doc/howtos/backend/exercise-basic-action [hunk 1]
```diff
@@ -8,7 +8,7 @@ Index: addons/openacademy/__manifest__.py
      'data': [
          # 'security/ir.model.access.csv',
          'templates.xml',
-+        'views/openacademy.xml',
++    'views/openacademy.xml',
```
Comment (pedrobaeza) [inline doc/howtos/backend/exercise-basic-action]: Incorrect change

**Comment by moylop260** [comment]
@mart-e @pedrobaeza Thanks for feedback. Now is fixed!


The `make build` now works without errors:

 - screen shot 2017-07-11 at 15 21 00

File: doc/reference/guidelines.rst [hunk 2]
```diff
@@ -163,7 +163,8 @@ To declare a record in XML, the **record** notation (using *<record>*) is recomm
 - Try to group the record by model. In case of dependencies between
   action/menu/views, this convention may not be applicable.
 - Use naming convention defined at the next point
-- The tag *<data>* is only used to set not-updatable data with ``noupdate=1``
```
Comment (mart-e) [inline doc/reference/guidelines.rst]: I am not sure about removing references to the `data` tag. It is still valid and, as we discussed in a previous issue, using data is still necessary for two noupdate clause in the same file

Comment (moylop260) [inline doc/reference/guidelines.rst]: You are right @mart-e 
I just updated this guideline using the comment from commit message https://github.com/odoo/odoo/commit/7c0e598c97cc48d8e55688c99db6d11ee23d9e46
`The tag <data> is now optional and should only be used if the same xml file contains both update and noupdate records.`

**Comment by mart-e** [comment]
Thanks, merged at ca82c77d48f

**Pull request closed without merging**